apps/frontend-manage/src/components/activities/overview/LiveQuizActions.tsx

```diff
@@ -152,6 +167,11 @@ function LiveQuizActions({
     isShared: liveQuiz.isShared,
   })
 
+  // Add activity log action at the beginning of the dropdown, leaving the first 3 buttons unchanged
+  const visibleActions = baseActions.slice(0, 3)
```

Comment:
The rendering logic for this part was already optimized before to be as consistent as possible and only show a corresponding dropdown if necessary. We should not manually add the activity log to the dropdown actions, but treat it the same as any other available action.
For this, it needs to be added to the useLiveQuizActions hook, where we conditionally determine when to render which action options (depending on the activity status and the user's permissions on the object)